GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/StatusVerbTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using GVFS.FunctionalTests.FileSystemRunners;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GVFS.FunctionalTests.Tests.EnlistmentPerFixture
+{
+    [TestFixture]
+    public class StatusVerbTests : TestsWithEnlistmentPerFixture
+    {
+        private enum ExpectedReturnCode
+        {
+            Success = 0,
+            ParsingError = 1,
+        }
+
+        [TestCase]
+        public void GitTrace()
+        {
+            Dictionary<string, string> environmentVariables = new Dictionary<string, string>();
+
+            this.Enlistment.Status(trace: "1");
+            this.Enlistment.Status(trace: "2");

```

Comment:
Do we need to verify anything for these?

Reply:
So `gvfs status` could fail to parse the `git status` it calls if the `GIT_TRACE` values are not removed.

Reply:
Looks like we are sending a `gvfs status` command.  Will that cause a `git status` to run as well?  I wasn't able to find that in the quick search I did.  Could you point me at that code?

Reply:
You're right. I thought it did, but it doesn't. I must call _something_, since `logPath` is filled with contents.

Reply:
Ok, I took a look at my logs, and we actually run `git config` to get the remote url:

```
07:52:55.681330 exec-cmd.c:236          trace: resolved executable dir: C:/Program Files/Git/mingw64/bin
07:52:55.684326 run-command.c:638       trace: run_command: C:/_git/ForTests2/src/.git/hooks/pre-command.exe config --local remote.origin.url --git-pid=16732
07:52:55.808825 git.c:477               trace: built-in: git config --local remote.origin.url
07:52:55.809326 run-command.c:638       trace: run_command: C:/_git/ForTests2/src/.git/hooks/post-command.exe config --local remote.origin.url --git-pid=16732 --exit_code=0
```

This is run by `GitProcess.GetOriginUrl()`.

Reply:
The output is interpreted [here in `Enlistment.cs`](https://github.com/Microsoft/VFSForGit/blob/master/GVFS/GVFS.Common/Enlistment.cs#L36-L47) and would fail if we didn't redirect `GIT_TRACE=1` (the output would not be a URL) or `GIT_TRACE=2` (`originResult.HasErrors` would be true).